health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/SelectingBoundariesDuplicate.js

```
@@ -102,53 +163,66 @@ const SelectingBoundariesDuplicate = ({ onSelect, formData, ...props }) => {
+                // }}
+                  hierarchyType={hierarchyType}
+                  lowest={lowestHierarchy}
+                  selectedData={selectedData}
+                  boundaryOptions={boundaryOptions}
+                  hierarchyData={boundaryHierarchyData} // Pass the fetched boundary hierarchy data
+                  isMultiSelect={"true"}
+                  restrictSelection={restrictSelection}
+                  onSelect={(value) => {
+                    handleBoundaryChange(value);
+                  }}
+              ></Wrapper>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)